cockroachcloud/create-your-cluster.md

```diff
@@ -27,11 +27,9 @@ To create and connect to a 30-day free {{ site.data.products.dedicated }} cluste
 
 In the **Cloud provider** section, select either **Google Cloud** or **AWS** as your preferred cloud provider.
 
-{{ site.data.products.db }} GCP clusters use [N2 standard](https://cloud.google.com/compute/docs/machine-types#n2_machine_types) machine types and [Persistent Disk storage](https://cloud.google.com/compute/docs/disks#pdspecs). AWS clusters use [M5 instance types](https://aws.amazon.com/ec2/instance-types/m5/#Product_Details) and [Elastic Block Store (EBS)](https://aws.amazon.com/ebs/features/). The IOPS associated with each node size in GCP is equal to 30 times the storage size, and the IOPS for AWS nodes is equal to 15 times the storage size.
+{{ site.data.products.db }} GCP clusters use [N2 standard](https://cloud.google.com/compute/docs/machine-types#n2_machine_types) machine types and [Persistent Disk storage](https://cloud.google.com/compute/docs/disks#pdspecs). AWS clusters use [M5 instance types](https://aws.amazon.com/ec2/instance-types/m5/#Product_Details) and [Elastic Block Store (EBS)](https://aws.amazon.com/ebs/features/). 
 
-{{site.data.alerts.callout_info}}
-If you created a {{ site.data.products.dedicated }} cluster before December 1, 2021, your cluster may have a different machine type, IOPS, and pricing. Your cluster will be transitioned to the current hardware configuration by the end of the month.
-{{site.data.alerts.end}}
+For GCP clusters, each GiB of storage costs $0.0012 per hour, and 30 IOPS per GiB are provisioned. For AWS clusters, each GiB of storage costs $0.0005 per hour, and 15 IOPS per GiB are provisioned at an additional cost of $0.0000196 per hour. 
```

Comment:
I would specify that they $0.000196 per hour is per IOPS. Just in case so there isn't confusion that 15 IOPS are worth that much, but it is each individual IOPS.